examples/travel-app/src/app/travel-form.tsx

```diff
@@ -67,37 +65,36 @@ export const LOADING_MESSAGES = [
   "Just one more thing",
 ];
 
+const date = new Date();
+const tomorrowDate = new Date(new Date().setDate(new Date().getDate() + 1));
```

Comment:
this is a bit confusing, we shouldn't need 3 new date objects right?

```suggestion
const tomorrowDate = new Date();
tomorrowDate.setDate(tomorrowDate.getDate() + 1);
```